base/cvd/README.md

```diff
@@ -0,0 +1,37 @@
+# Developing `cvd`
+
+`cvd` is a command line tool that is the interface to running Cuttlefish host
+executables from AOSP. It is deployed through the `cuttlefish-base` debian
+package, but can also be compiled and run on its own.
+
+## Compiling and running
+
+```sh
+bazel run cuttlefish:cvd -- reset -y
+```
+
+## Running the unit tests
+
+```sh
+bazel test '...'
+```
+
+## Color in compilation errors
+
+```sh
+bazel build --copt=-fdiagnostics-color=always '...'
+```
+or define a `.bazelrc` containing
```

Comment:
Can we just add this to the repo? Will it mangle the presubmit test logs?

Reply:
Looks like it is [standard practice](https://bazel.build/configure/best-practices#bazelrc-file) to define a project `.bazelrc` so I'll add it here.